cleverhans_tutorials/mnist_tutorial_cw.py

```diff
@@ -66,7 +66,7 @@ def mnist_tutorial_cw(train_start=0, train_end=60000, test_start=0,
     set_log_level(logging.DEBUG)
 
     # Get MNIST test data
-    X_train, Y_train, X_test, Y_test = data_mnist(train_start=train_start,
+    x_train, y_train, x_test, y_test = data_mnist(train_start=train_start,
```

Comment:
Why did you change the name of these variables?

Reply:
To be PEP8 compliant, I just did it mechanically since I was touching the file,

Reply:
Yeah, I think this is more correct. I still write a lot of non-PEP8 names as an old habit, but probably better to fix it.

Reply:
(Also, these variables aren't part of an API, so it won't break anyone's code)

Reply:
That makes a lot of sense, we should change the names in that case